snap/snapcraft.yaml

```diff
@@ -15,12 +15,11 @@ architectures:
 parts:
   brave:
     plugin: dump
-    source: https://github.com/brave/browser-laptop/releases/download/v0.24.0dev/brave_0.24.0_amd64.deb
+    source: https://github.com/brave/brave-browser/releases/download/v0.55.22/brave-browser_0.55.22_amd64.deb
     source-type: deb
     # Correct path to icon.
-    prepare: |
-      sed -i 's|Icon=brave|Icon=/usr/share/pixmaps/brave\.png|g' usr/share/applications/brave.desktop
-      echo StartupWMClass=brave >> usr/share/applications/brave.desktop
+    override-build: |
+      rm -rf etc/cron.daily/ &&  rm -rf usr/bin/brave-browser
```

Comment:
You're missing `snapcraftctl build` immediately after `override-build: |` so the dumped deb is never going to be pulled into the build. Add that line and it should work. Alternatively use `override-pull` and `snapcraftctl pull` before your `rm`

Reply:
@popey how am i looking now?